.github/workflows/interface_tests.yml

```diff
@@ -0,0 +1,23 @@
+name: Interface Tests
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - src/interfaces/coral_web/**
```

Comment:
[source](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore)